src/front/app/features/auth/components/LoginButton.integration.spec.js

```diff
@@ -11,11 +11,11 @@ describe('LoginButton', () => {
     beforeEach(() => {
         onClick = env.stub();
         text = 'test Button Text';
-        wrapper = mount(<LoginButton text={text} onLogin={ onClick } />);
+        wrapper = mount(<LoginButton text={text} onClick={ onClick } />);
     });
 
     it('should render with text', () => {
-        wrapper.find('.login-button').text().should.equal(text);
+        wrapper.find('.login-button').text().should.equal(`Sign up with ${text}`);
```

Comment:
Are you sure you've meant "Sign up"? "To sing up" is "to register" while "to sing in" is "to login".